<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="hw1819.cignitiglobal.com" name="Grouping" tests="2" failures="2" timestamp="12 Jul 2018 11:42:20 GMT" time="280.585" errors="0">
  <testcase name="TC_01_a_memberRegistration" time="127.571" classname="flynas.ios.uat.reg.TC01_a_memberRegistration">
    <failure type="java.lang.AssertionError" message="Step WaitForElementPresent  Failed with description Falied to locate element Register expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Step WaitForElementPresent  Failed with description Falied to locate element Register expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ctaf.support.HtmlReportSupport.onFailure(HtmlReportSupport.java:779)
	at com.ctaf.utilities.Reporter.failureReport(Reporter.java:91)
	at com.ctaf.accelerators.ActionEngine.waitForElementPresent(ActionEngine.java:755)
	at flynas.ios.workflows.RegistrationPage.register(RegistrationPage.java:41)
	at flynas.ios.uat.reg.TC01_a_memberRegistration.TC_01_a_memberRegistration(TC01_a_memberRegistration.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- TC_01_a_memberRegistration -->
  <testcase name="TC_01_b_verifyRegistrationFailurewithExistingEmailID" time="126.497" classname="flynas.ios.uat.reg.TC01_b_verifyRegistrationFailurewithExistingEmailID">
    <failure type="java.lang.AssertionError" message="Step WaitForElementPresent  Failed with description Falied to locate element Register expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Step WaitForElementPresent  Failed with description Falied to locate element Register expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ctaf.support.HtmlReportSupport.onFailure(HtmlReportSupport.java:779)
	at com.ctaf.utilities.Reporter.failureReport(Reporter.java:91)
	at com.ctaf.accelerators.ActionEngine.waitForElementPresent(ActionEngine.java:755)
	at flynas.ios.workflows.RegistrationPage.register(RegistrationPage.java:41)
	at flynas.ios.uat.reg.TC01_b_verifyRegistrationFailurewithExistingEmailID.TC_01_b_verifyRegistrationFailurewithExistingEmailID(TC01_b_verifyRegistrationFailurewithExistingEmailID.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- TC_01_b_verifyRegistrationFailurewithExistingEmailID -->
</testsuite> <!-- Grouping -->
